app/views/candidate_mailer/chase_reference.text.erb

```diff
@@ -1,9 +1,9 @@
-<%= @reference.name %> has not responded yet.
+<% if @application_form.show_new_reference_flow? %>
 
-You can add as many referees as you like to increase the chances of getting 2 references quickly.
+<%= render 'new_flow_chase_reference' %>
 
-You can also send one reminder to each referee from your account.
+<% else %>
 
-[Manage your references](<%= candidate_magic_link(@application_form.candidate) %>).
+<%= render 'old_flow_chase_reference' %>
 
-You cannot submit your application without 2 references. Courses can become full anytime - get your references as soon as possible.
+<% end %>
```

Comment:
Could now move this logic into the mailer class itself, and promote the partials into full templates?  (You can set the template to use by adding a `template_name` param to the `mail` method, see [docs](https://guides.rubyonrails.org/action_mailer_basics.html#mailer-views)).

Just a suggestion though, and might not be worth it given that we should be able to come back and delete all the old flow templates once the cycle has opened?

Reply:
@frankieroberto I thought a similar thing about the mailer template names but given everything in the candidate (and other) mailers call [this method](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/mailers/application_mailer.rb#L17-L20) I think it's a larger refactor.

Reply:
@steventux ah - didn't realise there was another level of indirection here...